### ISSUE TITLE
Update release notes for release 5.5

### ISSUE THREAD

**Issue by sheryjoe** [SCIInstitute/ShapeWorks#749]
Please add a statement for each new/improve/bug-fix included in this release.

**Comment by medakk** [comment]
Mesh changes:

```
* ShapeWorks now supports optimization directly on open meshes. See the two new use cases femur_mesh and lumps for an example of how to get started. This allows for large memory savings(9.2GB to 53MB in the femur use case)
```

**Comment by akenmorris** [comment]
Added @medakk 's changes and some others about Studio to docs/about/release-notes.md

**Comment by sheryjoe** [comment]
Great! Thanks, @medakk @akenmorris! Can't wait to see release notes from the rest of you @jadie1 @riddhishb @cchriste @archanasri @HeavenlyBerserker 

Also, I am thinking of having a "Release Graphical Abstract" to send with the announcement (SCI page, researchgate, linkIn, shapeworks-users) and have it on our (to-be) news page  -- **Any ideas are very much appreciated!!!!!**

**Comment by jadie1** [comment]
Here are my updates:
- Use case organization has been updated so downloaded data goes into one folder (ShapeWorks/Examples/Python/Data/) and use case output goes into another (ShapeWorks/Examples/Python/Output/). 
- Femur use case can now be run without grooming images. To groom with images use tag `--start_with_image_and_segmentation_data`, if this tag is not included images will not be used in grooming.
- All data sets on the portal now have the shape model output from running the use cases with a corresponding analyze.xml. Users can cd to the where the data is extracted and call `ShapeWorksStudio analyze.xml` to visualize these shape models. 
- A Python package for data augmentation has been added. (See ShapeWorks/docs/deep-learning/data-augmentation.md)
- A Python package for deep learning shape models from unsegmented images has been added. (See ShapeWorks/docs/deep-learning/deep-ssm.md)
- We added a new use case called deep_ssm that demonstrates data augmentation and deep learning on the femur data. (See ShapeWorks/docs/use-cases/deep-ssm-femur.md)

**Comment by sheryjoe** [comment]
> Added @medakk 's changes and some others about Studio to docs/about/release-notes.md

where did you add this?

**Comment by HeavenlyBerserker** [comment]
Changes:

   - Implemented Update Projection Cutting Planes Implementation
   - Removed the previous non-shadow implementation which did not guarantee
   non-violation of cutting planes.
   - Made each step of initialization and optimization constraint aware for
   the image domain, meaning that at no point, particles will violate
   constraints. This allows shadow particles to work with arbitrary cutting
   planes as well.
   - Added constraint aware raster scan (image domain) so that the starting
   point won’t violate the any constraints. This allows shadow particles to
   work with arbitrary cutting planes as well.
   - Added constraint aware particle splitting (image domain). The particle
   splitting only occurred towards a single quadrant, fixed. Particle
   splitting now shifts in a different direction for each particle in every
   domain.
   - Old constraints functionality (which wasn’t working) moved to the
   constraints class. Shadow particles implementation is now based on data
   stored by the constraints class but lives in
   itkOarticleOmegaGradientFunction.
   - Made applyConstraints (image domain) constraint aware. Snapping to
   surface won’t cause constraint violations, which was never observed but
   possible before.
   - Added buffer area of 1e-10 for plane constraints. This is necessary
   for shadow particles to work and provides numerical stability to the update
   projection implementation
   - Added femur_cut
   - Added ellipsoid_cut
   - Added adaptivity_mode, cutting_planes, and cutting_plane_counts to the
   parameter dictionary.
   - Added ShowCuttingPlanesOnImage to GroomUtils
   - Created unit tests for 2 cutting planes and for sphere.

Known issues:

   - Ellipsoid_cut with 32 particles does not work.
   - Tiny artifacts in the 2 cutting plane unit test.

**Comment by sheryjoe** [comment]
> * All data sets on the portal now have the shape model output from running the use cases with a corresponding analyze.xml. Users can cd to the where the data is extracted and call `ShapeWorksStudio analyze.xml` to visualize these shape models.

@jadie1 are the analyze.xml contain valid relative paths to particle files and distance transforms to launch Studio from the dataset location? Could you please double check?

**Comment by archanasri** [comment]
Consolidation of image-based and segmentation-based tools that creates a stable and reusable API making it much easier and more flexible for users to groom their datasets.
Includes a full complement of unit tests.

**Comment by sheryjoe** [comment]
Thanks all of your help with this! I just committed the release notes. Could you please have a look?

01799dd added release notes

**Comment by sheryjoe** [comment]
Addressed in #746.
